From 81df03e590a2fbf79d363910ae4edc581e09104e Mon Sep 17 00:00:00 2001
From: Stephen Warren <swarren@nvidia.com>
Date: Tue, 28 Jan 2014 21:50:09 -0700
Subject: [PATCH 01/13] pxe: support "devicetree" tag

pxe: support "devicetree" tag

The specification for extlinux.conf[1] states that "fdt" is an alias for
"devicetree". To date, U-Boot only implements "fdt". Rectify that.

[1] http://freedesktop.org/wiki/Specifications/BootLoaderSpec/

Signed-off-by: Stephen Warren <swarren@nvidia.com>
---
 common/cmd_pxe.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/common/cmd_pxe.c b/common/cmd_pxe.c
index c27ec35..4f00b1a 100644
--- a/common/cmd_pxe.c
+++ b/common/cmd_pxe.c
@@ -745,6 +745,7 @@ static const struct token keywords[] = {
 	{"append", T_APPEND},
 	{"initrd", T_INITRD},
 	{"include", T_INCLUDE},
+	{"devicetree", T_FDT},
 	{"fdt", T_FDT},
 	{"ontimeout", T_ONTIMEOUT,},
 	{"ipappend", T_IPAPPEND,},
-- 
1.8.5.3

